examples/next/pages/ui/components/authenticator/sign-in-totp-mfa/index.page.tsx

```diff
@@ -8,10 +8,16 @@ Amplify.configure(awsExports);
 
 export default function SignInTotpMfa() {
   const formFields = { setupTOTP: { QR: { totpIssuer: 'My Web App' } } };
+
   return (
     <>
       <Authenticator formFields={formFields}>
-        {({ signOut }) => <button onClick={signOut}>Sign out</button>}
+        {({ signOut, user }) => (
+          <>
+            <span>{user?.username}</span>
```

Comment:
Do we need the optional chaining? Shouldn't `user.username` always be defined if the `Authenticator` renders its children?